pkg/3scale/amp/operator/apicast_options_provider.go

```diff
@@ -183,6 +183,12 @@ func (a *ApicastOptionsProvider) stagingPodTemplateLabels() map[string]string {
 		labels[k] = v
 	}
 
+	if a.apimanager.Spec.Apicast.StagingSpec.Labels != nil {
```

Comment:
no need to check. When `a.apimanager.Spec.Apicast.StagingSpec.Labels` is  `nil`, `range` will handle correctly.

Reply:
Addressed with this [commit ](https://github.com/3scale/3scale-operator/pull/835/commits/a92f1f214c07bead0c9650c1dbbf8003b80a09ab)